src/library_fs.js

```diff
@@ -1347,6 +1347,13 @@ mergeInto(LibraryManager.library, {
       FS.createDefaultDirectories();
       FS.createDefaultDevices();
       FS.createSpecialDirectories();
+
+      FS.filesystems = {
```

Comment:
please also define the `filesystems` property on the object itself, with value `null`. That avoids the JS engines seeing a new property being added here, which can cause reoptimization.